app/config/parameters.yml.dist

```diff
@@ -48,6 +48,7 @@ parameters:
     env(ALGOLIA_FRONT_API_KEY): ~
     env(ALGOLIA_BACK_API_KEY):  ~
     env(ALGOLIA_BLACKLIST):     ~
+    env(ALGOLIA_INDEX_PREFIX):  'dev'
```

Comment:
`'dev'` => `'%kernel.environment%'`?

Reply:
actually, no.
Algolia indexes are already suffixed by the APP_ENV environment variable.

I introduced this new env variable so we can have differents indexes for each environment.
(staging has APP_ENV=prod, like production., but we want different indexes for staging & prod !)

Reply:
ok, was just asking ^^', thx!